addons/stock/models/stock_move.py

```diff
@@ -164,12 +164,23 @@ def _default_group_id(self):
     has_move_lines = fields.Boolean(compute='_compute_has_move_lines')
     package_level_id = fields.Many2one('stock.package_level', 'Package Level')
     picking_type_entire_packs = fields.Boolean(related='picking_type_id.show_entire_packs', readonly=True)
+    next_serial_number_name = fields.Char('Next Serial Number')
+    display_assign_sn_button = fields.Boolean(compute='_compute_display_assign_sn_button')
 
     @api.onchange('product_id', 'picking_type_id')
     def onchange_product(self):
         if self.product_id:
             self.description_picking = self.product_id._get_description(self.picking_type_id)
 
+    @api.depends('picking_type_id.use_create_lots', 'picking_type_id.use_existing_lots')
```

Comment:
product_id.tracking ?

Reply:
I set the depend on `has_tracking` (related to `product_id.tracking`), I don't know if it made any difference but I don't think.
``` python
@api.depends('has_tracking')
```